airbyte-integrations/connectors/source-auth0/source_auth0/manifest.yaml

```diff
@@ -0,0 +1,115 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+
+  requester:
+    type: HttpRequester
+    url_base: "{{ config['base_url'] }}"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['credentials']['access_token'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "#/definitions/requester"
+
+  base_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/retriever"
+
+  clients_stream:
+    $ref: "#/definitions/base_stream"
+    name: "clients"
+    primary_key: "client_id"
+    $parameters:
+      path: "clients"
+
+  users_stream:
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    # incremental_sync:
```

Comment:
@sajarin Full Refresh tests are passing successfully
![image](https://github.com/airbytehq/airbyte/assets/47392334/964231cd-afd8-495a-8b05-81ed6b346c63)


But the connection is getting lost while adding incremental sync,
Could you please check,
I've commented on incremental sync for ease :)